### PR TITLE
multiplexing edits

### PR DESCRIPTION
**Pull request by walkerlj0**
Minor updates to multiplexing doc

### CODE REVIEW

File: content/concepts/stream-multiplexing.md [hunk 1]
```diff
@@ -2,17 +2,21 @@
 title: Stream Multiplexing
 ---
 
-Stream Multiplexing (often abbreviated as "stream muxing") allows multiple independent logical streams to all share a common underlying transport medium.
+Stream Multiplexing (_stream muxing_) is a way of sending multiple streams of data over one communication link. It combines multiple signals into one unified signal so it can be transported 'over the wires', then it is demulitiplexed (_demuxed_) so it can be output and used by separate applications.
+
+//what type of multiplexing? Time-division? Frequency division because blocks?
```
Comment (mxinden) [inline content/concepts/stream-multiplexing.md:7]: Is this a leftover comment?

File: content/concepts/stream-multiplexing.md [hunk 2]
```diff
@@ -2,17 +2,21 @@
...
+
+This is done to share the transmission bandwidth available between multiple sources to make transmission more efficient.
```
Comment (mxinden) [inline content/concepts/stream-multiplexing.md:9]: Is this sentence needed? The paragraph below already extends on the motivation behind multiplexing.

File: content/concepts/stream-multiplexing.md (hunk 1 above)
Comment (walkerlj0) [reply content/concepts/stream-multiplexing.md:7]: Yes, wasn't actually ready to merge, want to touch on tomorrow @mxinden

**Review by walkerlj0 — COMMENTED** [review]
@mxinden @marten-seemann re-did this PR because I acidentally deleted a file in the last one when I meant to exclude it from the PR.

@mxinden I re-did the first paragraphs of stream-multiplcexing from the eyes of someone who has never heard of it before, idea was to give a high-level overview before digging into details, up to you if you think it's relevant.

**Review by marten-seemann — CHANGES_REQUESTED** [review]
I'm confused now. @walkerlj0, why did you close #134 and add the changes to this PR? It makes it hard for me to see how my review comments have been incorportated.

**Pull request closed without merging**